tests/cli.rs

```diff
@@ -9,29 +9,68 @@ use tempfile::NamedTempFile;
 // - Figure out how to mock tmux or use optional env var in main when testing
 // for its presence.
 
+const LONG_HELP: &'static str = r#"rmuxinator 0.1.0
```

Comment:
Is there no way to pull this in from cargo? (I'll look into this myself, too.)

Reply:
You can get the [help message directly from `clap`](https://docs.rs/clap/2.33.1/clap/struct.App.html#method.write_help) actually, but I'm not too sure how best to provide the tests with a way to access that. Maybe provide a `pub fn` for building the `clap::App` that is used by `parse_args` to parse the arguments and can additionally be used by the tests to get the short and long help messages? I'll think on it a bit to see what I settle on.